rust/lance-index/src/vector/flat.rs

```diff
@@ -34,56 +23,19 @@ fn distance_field() -> ArrowField {
 
 #[instrument(level = "debug", skip_all)]
 pub async fn flat_search(
```

Comment:
NIT: this method name doesn't match what it does since this PR

Reply:
ok, fixed.